### PR TITLE
[#176014693] Add purge cdn in CI/CD pipeline

### PR DESCRIPTION
**Pull request by infantesimone**
### Description

This PR adds a step in CI/CD pipeline to purge the _cdn_ as the last deployment step:

```yml
       - task: AzureCLI@1
            displayName: 'Purge CDN'
            inputs:
              azureSubscription: '$(PRODUCTION_AZURE_SUBSCRIPTION)'
              scriptLocation: inlineScript
              inlineScript: |
                call az cdn endpoint purge -g $(PRODUCTION_RESOURCE_GROUP_NAME) -n $(PRODUCTION_CDN_ENDPOINT) --profile-name $(PRODUCTION_CDN_PROFILE_NAME) --content-paths "/*"

```

It is required to set the following variables related to the pipeline on Azure DevOps:
- PRODUCTION_RESOURCE_GROUP_NAME
- PRODUCTION_CDN_ENDPOINT
- PRODUCTION_CDN_PROFILE_NAME

### CODE REVIEW

**Review by AleDore — COMMENTED** [review]
LGTM , @balanza ?

File: .devops/deploy-pipelines.yml [hunk 1]
```diff
@@ -138,3 +138,11 @@ stages:
...
               containerName: '$web'
             displayName: Deploy to container blob
+
+          - task: AzureCLI@1
+            displayName: 'Purge CDN'
+            inputs:
+              azureSubscription: '$(PRODUCTION_AZURE_SUBSCRIPTION)'
+              scriptLocation: inlineScript
+              inlineScript: |
+                call az cdn endpoint purge -g $(RESOURCE_GROUP_AZURE) -n $(ENDPOINT_AZURE) --profile-name $(PROFILE_NAME_CDN_AZURE) --content-paths "/*"
```
Comment (balanza) [inline .devops/deploy-pipelines.yml:148]:
```suggestion
                call az cdn endpoint purge -g $(RESOURCE_GROUP_AZURE) -n $(ENDPOINT_AZURE) --profile-name $(PROFILE_NAME_CDN_AZURE) --content-paths "/*"

```

Comment (balanza) [inline .devops/deploy-pipelines.yml:148]: We can rename variables to be the same we use in other pipelines:
```suggestion
	                call az cdn endpoint purge -g $(PRODUCTION_RESOURCE_GROUP_NAME) -n $(PRODUCTION_CDN_ENDPOINT) --profile-name $(PRODUCTION_CDN_PROFILE_NAME) --content-paths "/*"
```

Do they make sense?

Comment (infantesimone) [reply .devops/deploy-pipelines.yml:148]: ok 👍

**Comment by balanza** [comment]
Great job! We really need it.

**Pull request merged**